Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: The error message indicates the radio field is required. With radio `input` elements, you can set `required` on each `input` and browsers will only require a user to choose one of them.
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: A `button` is not part of this component.
diff --git a/output/Radio Group with Validation Error/2/f0f39df5-8ca6-4a7b-9626-f1e449e5962a.html b/output/Radio Group with Validation Error/2/f0f39df5-8ca6-4a7b-9626-f1e449e5962a.html
index 0dbca18..5d5e866 100644
--- a/output/Radio Group with Validation Error/2/f0f39df5-8ca6-4a7b-9626-f1e449e5962a.html	
+++ b/output/Radio Group with Validation Error/2/f0f39df5-8ca6-4a7b-9626-f1e449e5962a.html	
@@ -1,10 +1,20 @@
-<form>
-  <fieldset>
-    <legend>Choose a Theme:</legend>
-    <label><input type="radio" name="theme" value="light"> Light</label><br>
-    <label><input type="radio" name="theme" value="dark"> Dark</label><br>
-    <label><input type="radio" name="theme" value="high-contrast"> High Contrast</label><br>
-    <span style="color: red; display: block;">You must choose a theme to proceed</span>
-  </fieldset>
-  <button type="submit">Submit</button>
-</form>
\ No newline at end of file
+<fieldset aria-describedby="theme-error">
+  <legend>Choose a Theme:</legend>
+  <span id="theme-error">You must choose a theme to proceed</span>
+  <label for="light">
+    <input type="radio" id="light" name="theme" value="light"
+           required aria-invalid="true">
+    Light
+  </label>
+  <label for="dark">
+    <input type="radio" id="dark" name="theme" value="dark"
+           required aria-invalid="true">
+    Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" id="high-contrast" name="theme"
+           value="high-contrast"
+           required aria-invalid="true">
+      High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
